index.js

```diff
@@ -460,7 +460,7 @@ var ab2obj = function (ab) {
 
 // convert a hex string to an array buffer
 var hex2ab = function (s) {
-  return new Uint8Array(new Buffer(s, 'hex'))
+  return new Uint8Array(new Buffer(typeof s !== 'string' ? '' : s, 'hex'))
```

Comment:
Since hex2ab is never supposed to take a string as input, please just return null or throw an error if `typeof s !== 'string`.